utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -312,6 +312,13 @@ int main(int argc, char **argv)
     input_second = static_cast<Rpp8u *>(calloc(inputBufferSize, 1));
     output = static_cast<Rpp8u *>(calloc(outputBufferSize, 1));
 
+    Rpp32f *rowRemapTable = (Rpp32f*) calloc(ioBufferSize, sizeof(Rpp32f));
```

Comment:
Please use a static case here
Also would suggest to allocate only for remap case

You can declare the pointer, but allocate it only for remap cases

Reply:
Made this change for HOST too